daprdocs/content/en/operations/configuration/increase-request-size.md

```diff
@@ -1,27 +1,31 @@
 ---
 type: docs
-title: "How-To: Handle large http body requests"
-linkTitle: "HTTP request body size"
+title: "How-To: Handle larger body requests"
+linkTitle: "Request body size"
 weight: 6000
 description: "Configure http requests that are bigger than 4 MB"
 ---
 
-By default, Dapr has a limit for the request body size, set to 4MB. You can change this by defining:
-- The `dapr.io/http-max-request-size` annotation, or
-- The `--dapr-http-max-request-size` flag.
+{{% alert title="Note" color="primary" %}}
+The existing flag/annotation `dapr-http-max-request-size` has been deprecated and replaced with the below. It will be removed in the next release.
```

Comment:
```suggestion
The flag/annotation `dapr-http-max-request-size` has been deprecated and replaced with the  `max-body-size` flag/annotation.
```

Reply:
@mikeee - Can you update the table in this article https://v1-16.docs.dapr.io/operations/support/breaking-changes-and-deprecations#Announced%20deprecations which is the list of deprecations. You can add announcement date of 1.14.0 and removal in 1.17.0. This is about preventing comments hidden in the docs about when etc.